docs/reference/ingest/processors/fingerprint.asciidoc

```diff
@@ -0,0 +1,89 @@
+[role="xpack"]
+[testenv="basic"]
+[[fingerprint-processor]]
+=== Fingerprint processor
+++++
+<titleabbrev>Fingerprint</titleabbrev>
+++++
+
+Calculates a hash that can be used for content fingerprinting based on the
+content of the specified fields in a document.
+
+[[fingerprint-options]]
+.Fingerprint Options
+[options="header"]
+|======
+| Name               | Required | Default       | Description
+| `fields`           | yes      | n/a           | Array containing the names of the document
+fields whose content will be used to compute the fingerprint
+| `target_field`     | no       | `fingerprint` | Output field for the fingerprint.
+| `salt`             | no       | <none>        | Salt value that will be applied to every
+fingerprint calculated by this processor
+| `method`           | no       | `SHA-1`       | The cryptographic hash function used to
+compute the fingerprint. Must be one of `MD5`, `SHA-1`, `SHA-256`, or `SHA-512`.
+| `ignore_missing`   | no       | `false`       | If true, any missing elements in `fields`
+will be ignored when computing the fingerprint. If all fields are missing, the processor
+silently exits without modifying the document.
```

Comment:
Minor nit: I'd make `true` a literal and reword the first sentence for active voice.
```suggestion
| `ignore_missing`   | no       | `false`       | If `true`, the processor
ignores any missing `fields`. If all fields are missing, the processor silently
exits without modifying the document.
```